engines/base/src/test/java/org/alfresco/transform/base/clients/SfsClient.java

```diff
@@ -71,17 +72,17 @@ public static String uploadFile(final String fileToUploadName, final String sfsB
             sfsBaseUrl+"/alfresco/api/-default-/private/sfs/versions/1/file");
         post.setEntity(MultipartEntityBuilder
             .create()
-            .setMode(HttpMultipartMode.BROWSER_COMPATIBLE)
+            .setMode(HttpMultipartMode.LEGACY)
```

Comment:
Based on official documentation description for both of these options, they should be the same.